salt/metalk8s/orchestrate/bootstrap_with_master.sls

```diff
@@ -27,6 +27,14 @@ Bootstrap control plane:
     - pillar:
         registry_ip: {{ pillar.get('registry_ip') }}
 
+Wait for kube-apiserver to be ready:
```

Comment:
Instead of a custom procedure (`metalk8s.wait_apiserver`), couldn't this be `kubernetes.ping` with a `retry`? https://docs.saltstack.com/en/latest/ref/states/requisites.html#retrying-states

Reply:
So yes, it could, but IIRC, we introduced a `cri.wait_container` procedure specifically to avoid using `retry` from Salt: rationale was that we don't like seeing error logs during the bootstrap if it's just something we are retrying.